.circleci/config.yml

```diff
@@ -31,26 +26,29 @@ jobs:
 
   postgresql-ssl:
     docker:
-      - image: cossacklabs/android-build
+      - image: cossacklabs/ci-py-go-themis:latest
       - image: cossacklabs/postgresql-ssl:11-1
         environment:
           POSTGRES_PASSWORD: test
           POSTGRES_USER: test
           POSTGRES_DB: test
     environment:
       FILEPATH_ERROR_FLAG: /tmp/test_fail
-      VERSIONS: 1.13.15 1.14.9 1.15.2
-      TEST_DB_PORT: 5432
-      GOPATH_FOLDER: gopath
+      GO_VERSIONS: 1.13.15 1.14.9 1.15.2
+      TEST_DB_PORT: "5432"
       TEST_RANDOM_DATA_FOLDER: /tmp/test_data
       TEST_TLS: "on"
-      GOPATH: /home/user/gopath
     steps:
-    # prepare
+      # prepare
       - checkout
-      - run: .circleci/prepare.sh
-      - run: sudo apt-get install -y postgresql-client
-      - run: pg_isready -U${POSTGRES_USER} -d${POSTGRES_DB} -h127.0.0.1
+      # database in neighbor container need some time to start
+      # wait for the DB to be reachable, up to 15 pings with 1 second interval
+      - run: |
+          set +eo pipefail;
+          for try in {0..15}; do
+            pg_isready -U${POSTGRES_USER} -d${POSTGRES_DB} -h127.0.0.1; [ $? -eq 0 ] && exit 0 || sleep 1;
+          done;
+          exit 1;
```

Comment:
You don't really need semicolons here, it's a truly multiline script if you use `|` to start the block in YAML.

Also, the `[ $? -eq 0 ]` is redundant, you can use `condition && consequent || alternate` just fine. (The `$?` value is preserved in the clauses, if you need it. Also, using this form removes the need to `set +e`.)

Reply:
Do you think it's good idea to include one little script in docker container and then call it here and in other jobs?
The script would execute passed command passed amount of times. Just to make `config.yml` more readable, like this:
`try_few_times --count 15 --sleep 1 pg_isready -U${POSTGRES_USER} -d${POSTGRES_DB} -h127.0.0.1`

Reply:
> Do you think it's good idea to include one little script in docker container and then call it here and in other jobs?

In that case I'd rather have it as

```yaml
- run: .circleci/check_postgresql.sh
```

which is much more readable.